docs/getting_started/installation/cpu/x86.inc.md

```diff
@@ -1,19 +1,18 @@
 # --8<-- [start:installation]
 
-vLLM initially supports basic model inferencing and serving on x86 CPU platform, with data types FP32, FP16 and BF16.
+vLLM supports basic model inferencing and serving on x86 CPU platform, with data types FP32, FP16 and BF16.
 
```

Comment:
I don't think this is neccessary to include explicitly, the structure of the document makes it clear that you can use pre-built images or build from source

```suggestion
```